txnkv/transaction/txn.go

```diff
@@ -467,6 +467,9 @@ func (txn *KVTxn) InitPipelinedMemDB() error {
 	pipelinedMemDB := unionstore.NewPipelinedMemDB(func(ctx context.Context, keys [][]byte) (map[string][]byte, error) {
 		return txn.snapshot.BatchGetWithTier(ctx, keys, txnsnapshot.BatchGetBufferTier)
 	}, func(generation uint64, memdb *unionstore.MemDB) (err error) {
+		if atomic.LoadUint32((*uint32)(&txn.committer.ttlManager.state)) == uint32(stateClosed) {
+			return errors.New("ttl manager is closed")
```

Comment:
Should we also print a `WARN` log here?

Reply:
This will lead to query failure, and the log in TiDB be like.

<img width="1126" alt="image" src="https://github.com/tikv/client-go/assets/9587680/afd3f9da-5bb9-4fc6-a247-4545d7f71f19">